pages/07-solid-gauge-chart/simple.page.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Highcharts from "highcharts";
+import { omit } from "lodash";
+
+import "highcharts/highcharts-more";
```

Comment:
This is imported by `useChartSettings`, activatable with a prop. Let's use that approach as it emulates async load. We can add the solid-gauge module support in there, too.